paimon-common/src/main/java/org/apache/paimon/predicate/LeafPredicate.java

```diff
@@ -69,7 +69,7 @@ public DataType type() {
         return type;
     }
 
-    public int index() {
+    public int fieldIndex() {
```

Comment:
do not modify this, compute engines rely on this api.

Reply:
done